src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -109,10 +103,7 @@ function mapStateToProps() {
   return (state: RootState): StoreProps => ({
     storesLoaded: storesLoadedSelector(state),
     stores: sortedStoresSelector(state),
-    statOrder: state.dimApi.settings.loStatSortOrder.map((hash) => statHashToType[hash]),
-    assumeMasterwork: state.dimApi.settings.loAssumeMasterwork,
-    minimumPower: state.dimApi.settings.loMinPower,
-    minimumStatTotal: state.dimApi.settings.loMinStatTotal,
+    settings: settingsSelector(state),
```

Comment:
Sorry what I was suggesting was something like:

```
const { loAssumeMasterwork, loMinPower, loMinStatTotal, loStatorder } = settingsSelector(state);
```

Reply:
Ah my bad, should be fixed now.